packages/@react-spectrum/searchwithin/stories/SearchWithin.stories.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {Item, Picker} from '@react-spectrum/picker';
+import React from 'react';
+import {SearchField} from '@react-spectrum/searchfield';
+import {SearchFieldProps} from '@react-types/searchfield';
+import {SearchWithin} from '../';
+import {SpectrumPickerProps} from '@react-types/select';
+import {SpectrumSearchWithinProps} from '@react-types/searchwithin';
+import {storiesOf} from '@storybook/react';
+
+storiesOf('SearchWithin', module)
+  .add(
+    'Default',
+    () => render({})
+  ).add(
+    'isDisabled: true',
+    () => render({isDisabled: true})
+  ).add(
+    'isRequired: true',
+    () => render({isRequired: true})
+  ).add(
+    'isRequired: true, necessityIndicator: label',
+    () => render({isRequired: true, necessityIndicator: 'label'})
+  )
+  .add(
+    'isRequired: false, necessityIndicator: label',
+    () => render({isRequired: false, necessityIndicator: 'label'})
+  ).add(
+    'custom width',
+    () => render({width: 300})
+  ).add(
+    'labelPosition: side',
+    () => render({labelPosition: 'side'})
+  ).add(
+    'auto focus',
+    () => render({}, {autoFocus: true})
+  );
+
```

Comment:
odd request, but can you render one where someone entered the two elements in reverse? it should still render correctly (which we can visually handle with css `order` I'm not sure what to do about the tab order though in that case, might require a bit of experimentation and I'm ok with doing it as followup. If we can't figure out the tab order, then we'll need to call it out in the docs like we do for our triggers
```
    <SearchWithin label="Search" {...props}>
      <Picker defaultSelectedKey="all" {...pickerProps}>
        <Item key="all">All</Item>
        <Item key="campaigns">Campaigns</Item>
        <Item key="audiences">Audiences</Item>
        <Item key="tags">Tags</Item>
      </Picker>
      <SearchField placeholder="Search" {...searchFieldProps} />
    </SearchWithin>
```

Reply:
one other story, where just the scope picker is disabled as well

Reply:
Handled this with `order` now and added a story. Not sure how to handle the tab order so for now I just updated the docs to include `The SearchField element should be the first child passed into the SearchWithin`. I think this is probably how we want to handle this though. Let me know if you think this should be changed.

Related: https://www.w3.org/TR/css-flexbox-1/#order-accessibility